config/environments/production.rb

```diff
@@ -17,26 +17,17 @@
   config.i18n.fallbacks = true
 
   config.active_support.deprecation = :notify
-<<<<<<< HEAD
+
+  config.action_mailer.default_url_options = { host: 'myflixapp.herokuapp.com'}
```

Comment:
Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.